docs/examples/ratify-high-availability.md

```diff
@@ -0,0 +1,141 @@
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: dapr-redis
+spec:
+  type: state.redis
+  version: v1
+  metadata:
+  - name: redisHost
+    value: redis-master:6379
+  - name: redisPassword
+    secretKeyRef:
+      name: redis
+      key: redis-password
+  - name: primaryEncryptionKey
+    secretKeyRef:
+      name: ratify-dapr-signing-key
+      key: signingKey
+auth:
+  secretStore: kubernetes
+EOF
+
+kubectl apply -f dapr-redis.yaml -n gatekeeper-system
+```
+
+## Install Ratify
+```bash
+# download the notary verification certificate
+curl -sSLO https://raw.githubusercontent.com/deislabs/ratify/main/test/testdata/notary.crt
+helm install ratify \
+    ratify/ratify --atomic \
+    --namespace gatekeeper-system \
+    --set-file notaryCert=./notary.crt \
+    --set featureFlags.RATIFY_CERT_ROTATION=true \
+    --set featureFlags.RATIFY_DAPR_CACHE_PROVIDER=true \
+    --set replicaCount=3 \
+	--set provider.cache.type="dapr" \
+	--set provider.cache.name="dapr-redis"
+```
+
+## See Ratify in action
+
+- Deploy a `demo` constraint.
+```
+kubectl apply -f https://deislabs.github.io/ratify/library/default/template.yaml
+kubectl apply -f https://deislabs.github.io/ratify/library/default/samples/constraint.yaml
+```
+
+Once the installation is completed, you can test the deployment of an image that is signed using Notary V2 solution.
+
+- This will successfully create the pod `demo`
+
+```bash
+kubectl run demo --image=ghcr.io/deislabs/ratify/notary-image:signed
+kubectl get pods demo
+```
+
+Optionally you can see the output of the pod logs via: `kubectl logs demo`
+
+- Now deploy an unsigned image
+
+```bash
+kubectl run demo1 --image=ghcr.io/deislabs/ratify/notary-image:unsigned
+```
+
+You will see a deny message from Gatekeeper denying the request to create it as the image doesn't have any signatures.
+
+```bash
+Error from server (Forbidden): admission webhook "validation.gatekeeper.sh" denied the request: [ratify-constraint] Subject failed verification: wabbitnetworks.azurecr.io/test/net-monitor:unsigned
+```
+
+You just validated the container images in your k8s cluster!
+
+## Uninstall Ratify
+Notes: Helm does NOT support upgrading CRDs, so uninstalling Ratify will require you to delete the CRDs manually. Otherwise, you might fail to install CRDs of newer versions when installing Ratify.
+```bash
+kubectl delete -f https://deislabs.github.io/ratify/library/default/template.yaml
+kubectl delete -f https://deislabs.github.io/ratify/library/default/samples/constraint.yaml
+helm delete ratify --namespace gatekeeper-system
+kubectl delete crd stores.config.ratify.deislabs.io verifiers.config.ratify.deislabs.io certificatestores.config.ratify.deislabs.io
+helm delete redis --namespace gatekeeper-system
+helm delete dapr --namespace dapr-system
+kubectl delete Component dapr-redis -n gatekeeper-system
+kubectl delete Secret ratify-dapr-signing-key -n gatekeeper-system
+helm delete gatekeeper -n gatekeeper-system
+```
+
+
+
```

Comment:
Yeah that's a good point. I wasn't sure if we wanted to do this and instead keep this as a complete standalone guide since the quickstart has instructions on installing GK and ratify only and here we have new dependencies on Dapr and Redis that require a specific order. @susanshi thoughts?

Reply:
I prefer the quick start being basic steps. Maybe we can add a link , "ratify can be be configured with high availability, read more at Link"